src/containerapp/azext_containerapp/_params.py

```diff
@@ -49,6 +49,40 @@ def load_arguments(self, _):
+        c.argument('http_retry_max', type=int, options_list=['--http-retry'], help='Specifies the allowed number of retries. Default: 3.')
+        c.argument('http_retry_delay_in_milliseconds', type=int, options_list=['--http-delay'], help='Specifies the base interval between retries. Default: 1000.')
+        c.argument('http_retry_interval_in_milliseconds', type=int, options_list=['--http-interval'], help='Specifies the maximum interval between retries. Default: 10000.')
+        c.argument('http_retry_status_codes', nargs='*', options_list=['--http-codes'], help='A retry will be attempted if the response status code matches any status code in this list.')
+        c.argument('http_retry_errors', nargs='+', options_list=['--http-errors'], help='A retry will be attempted if the response error message matches any error in this list. Default: 5xx')
+
+    with self.argument_context('containerapp resiliency', arg_group='TCP Retry Policy') as c:
+        c.argument('tcp_retry_max_connect_attempts', type=int, options_list=['--tcp-retry'], help='The maximum number of unsuccessful connection attempts that will be made before giving up. Default: 3.')
+
+    with self.argument_context('containerapp resiliency', arg_group='TCP Connection Pool Policy') as c:
+        c.argument('tcp_connection_pool_max_connections', type=int, options_list=['--tcp-connections'], help='The maximum number of connections that will be made to the container app. Default: 1024.')
+
+    with self.argument_context('containerapp resiliency', arg_group='HTTP Connection Pool Policy') as c:
+        c.argument('http_connection_pool_http1_max_pending_req', type=int, options_list=['--http1-pending'], help='The maximum number of pending requests that will be allowed to the container app. Default: 1024.')
+        c.argument('http_connection_pool_http2_max_req', type=int, options_list=['--http2-parallel'], help='The maximum number of parallel requests that will be made to the container app. Default: 1024.')
+
+    with self.argument_context('containerapp resiliency', arg_group='Circuit Breaker Policy') as c:
+        c.argument('circuit_breaker_consecutive_errors', type=int, options_list=['--cb-consecutive-error'], help='The number of consecutive server-side error responses (for HTTP traffic, 5xx responses; for TCP traffic, failure to respond PONG; etc.) before a consecutive 5xx ejection occurs. Default: 5.')
+        c.argument('circuit_breaker_interval', type=int, options_list=['--cb-interval'], help='The time interval in seconds between ejection analysis sweeps. This can result in both new ejections as well as hosts being returned to service. Default: 10.')
+        c.argument('circuit_breaker_max_ejection', type=int, options_list=['--cb-max-ejection'], help='The maximum % of container app replicas that can be ejected. Defaults to 50% but will eject at least one host regardless of the value. Default: 50.')
```

Comment:
I want to ask what not add `default=50`? Currently if not input `--cb-max-ejection`, without `--default`, which value will be used?